api/src/main/java/io/opentelemetry/metrics/MeterProvider.java

```diff
@@ -26,7 +26,7 @@
  * @since 0.1.0
  */
 @ThreadSafe
-public interface MeterRegistry {
```

Comment:
Please comment that this is called `Provider` for consistency with other languages but it is not loaded using reflection (ServiceLoader)